### DISCUSSION TITLE
Change Navbar Logo

### DISCUSSION THREAD

**Discussion by tylerwmarrs** [apache/airflow#14485] [Q&A]
What is the best approach to change the Navbar logo without having to create an entirely new theme? I looked at the source code and found that the logo is an embedded SVG. I would like to change the logo so that I can more easily differentiate between production, staging, and development environments beyond just seeing the URL difference. Any guidance here is appreciated!

Here is where I see the logo:
https://github.com/apache/airflow/blob/master/airflow/www/templates/appbuilder/navbar.html#L32

**Comment by turbaszek** [comment]
You can fork the project, change the logo in source code and use your own version 😉 Other solution would be to implement a feature that would for example add a banner on top of the page (for example green for dev env).

**Reply by tylerwmarrs** [reply]
I'm trying to find a solution with the least amount of effort in regard to maintenance and implementation. Is creating a custom Jinja2 theme the common approach taken? I like the banner approach that you mentioned. Will you please guide me to a resource or steps to take in order to implement the banner?

**Reply by turbaszek** [reply]
> Will you please guide me to a resource or steps to take in order to implement the banner?

I would suggest starting with creating an feature request:
https://github.com/apache/airflow/issues/new/choose

so we can discuss how this can be done 👌 

As a instant solution I would consider experimenting with setting `navbar_color` and `instance_name` in `airflow.cfg`:
<img width="1680" alt="Screenshot 2021-02-26 at 21 12 57" src="https://user-images.githubusercontent.com/9528307/109350209-6a03b000-7877-11eb-8936-105d3a798919.png">
